backend/src/main/java/com/capstone/moneytree/service/impl/DefaultUserService.java

```diff
@@ -310,4 +297,33 @@ public boolean compareDigests(String plainPassword, String encryptedPassword) {
    public String getBucketName() {
       return bucketName;
    }
+
+   private String getSpringProfile(String profile) {
```

Comment:
Can you refactor this into its own class so that we use it elsewhere later? We can even put those in the application properties.

Reply:
That is good, it is considered a refactoring activity too for the sprint

Reply:
Should we keep it like it is? I think it`d be better if we set this in the properties. Less code.